packages/samples/react/src/components/input-radio/partials/variants.tsx

```diff
@@ -4,17 +4,19 @@ import { InputRadioCases } from './cases';
 
 import type { Components } from '@public-ui/components';
 import { SampleColumns } from '../../SampleColumns';
+import { KolHeading } from '@public-ui/react';
+
 export const InputRadioVariants = forwardRef<HTMLKolInputRadioElement, Components.KolInputRadio>(function InputRadioVariant(props, ref) {
 	return (
 		<SampleColumns>
-			<fieldset>
```

Comment:
Das hier ist der "eigentliche" Fix, der so auch in V2 Anwendung findet: Die doppelten, ineinander verschachtelten Fieldsets haben dazu geführt, dass die "{n} von {n}"-Ausgabe nicht funktioniert hat.

Reply:
Oje 💥